msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -504,6 +497,33 @@ Your app manifest must have the `webApplicationInfo` property to receive the mee
 }
  ```
 
+### Example of getting meeting `MeetingStartEndEventvalue`
+
+The bot receives event through the `OnEventActivityAsync` handler. To deserialize the json payload, a model object is introduced to get the metadata of a meeting. The metadata of a meeting is in the `value` property in the event payload. The `MeetingStartEndEventvalue` model object is created, whose member variables correspond to the keys under the `value` property in the event payload.
+     
+> [!NOTE]      
+> * Get meeting ID from `turnContext.ChannelData`.    
+> * Do not use conversation ID as meeting ID.     
+> * Do not use meeting ID from meeting events payload `turncontext.activity.value`.
+
+The following code shows how to capture the metadata of a meeting that is `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start/end event:
```

Comment:
how to capture a meeting's metadata such as `MeetingType`, `Title`, `Id`, `JoinUrl`, `StartTime`, and `EndTime` from a meeting start/end event:

Reply:
Done